WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsViewModel.swift

```diff
@@ -576,40 +576,14 @@ private extension SiteStatsInsightsViewModel {
         let periodSummary = periodStore.getSummary()
         updateMostRecentChartData(periodSummary)
 
-        let sparklineData: [Int] = makeSparklineData(countKey: \StatsSummaryData.likesCount)
-        let likesData = SiteStatsInsightsViewModel.intervalData(periodSummary, summaryType: .likes)
-
-        return StatsTotalInsightsData(count: likesData.count, difference: likesData.difference, percentage: likesData.percentage, sparklineData: sparklineData)
+        return StatsTotalInsightsData.createTotalInsightsData(periodStore: periodStore, statsSummaryType: .likes)
```

Comment:
I think this method could be named `createLikesTotalInsightsData`, like we have `createCommentsTotal...` used for the method below.

Reply:
Thanks!  I was trying to do something generic (thanks for noticing comments data) + I've updated with the generic intent I was hoping for originally